azureLogIngestion/adapter/__snapshots__/index.test.ts.snap

```diff
@@ -1,5 +1,111 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
+exports[`Adapter processes AppPerformanceCounters as metrics 1`] = `Array []`;
+
+exports[`Adapter processes AppPerformanceCounters as metrics 2`] = `
+Array [
+  CountMetric {
+    "attributes": Object {
+      "appRoleInstance": "{appRoleInstance Id}",
+      "appRoleName": "{appRoleName}",
+      "billedSize": 815,
+      "clientIp": "0.0.0.0",
+      "clientType": "PC",
+      "hostInstanceId": "{hostInstanceId}",
+      "instance": "??APP_WIN32_PROC??",
+      "processId": "4608",
+      "resourceId": "/SUBSCRIPTIONS/{subscriptionId}/RESOURCEGROUPS/{functionAppName}/PROVIDERS/MICROSOFT.INSIGHTS/COMPONENTS/{resourceGroupName}",
+      "sdkVersion": "af_azwapccore:2.16.0-18277",
+      "timestamp": 1621887508465,
+    },
+    "interval.ms": undefined,
```

Comment:
Is there a default interval for performance counters we can set here?

Reply:
Let's go with the 10000 you'd suggested as the default. Counters might have an interval property, but it's in the form of `\d+[ms|s|m]` (and probably typical cron values as well).

Reply:
I'm going to add a conversion method for these, but for right now I'm using the 10000 figure.